features/gebruik_in_lopende_tekst.feature

```diff
@@ -34,138 +34,126 @@ Rule: gebruikInLopendeTekst voor een persoon zonder adellijke titel of predicaat
-    | voorvoegsel met hoofdletters  | V        | P           | Jo Rene   | Op Den      | Berghe        | Van Der             | Broeck                | mevrouw Van Der Broeck               |
-    | voorvoegsel met hoofdletters  | O        | P           | Jo Rene   | Op Den      | Berghe        | Van Der             | Broeck                | J.R. Van Der Broeck                  |
-    | voorvoegsel begint met accent | V        | V           | Jo Rene   | in het      | Zonnetje      | 't                  | Wolkje                | mevrouw 't Wolkje-in het Zonnetje    |
-    | voorvoegsel met hoofdletters  | V        | V           | Jo Rene   | Op Den      | Berghe        | Van Der             | Broeck                | mevrouw Van Der Broeck-Op Den Berghe |
-    | voorvoegsel met hoofdletters  | O        | V           | Jo Rene   | Op Den      | Berghe        | Van Der             | Broeck                | J.R. Van Der Broeck-Op Den Berghe    |
-    | voorvoegsel in kleine letters | V        | N           | Jo Rene   | in het      | Zonnetje      | 't                  | Wolkje                | mevrouw In het Zonnetje-'t Wolkje    |
-    | voorvoegsel met hoofdletters  | V        | N           | Jo Rene   | Op Den      | Berghe        | Van Der             | Broeck                | mevrouw Op Den Berghe-Van Der Broeck |
-    | voorvoegsel met hoofdletters  | O        | N           | Jo Rene   | Op Den      | Berghe        | Van Der             | Broeck                | J.R. Op Den Berghe-Van Der Broeck    |
+    Gegeven de persoon met burgerservicenummer '000000164' heeft de volgende gegevens
+      | naam                                 | waarde          |
+      | geslachtsaanduiding (04.10)          | <geslacht>      |
+      | voornamen (02.10)                    | <voornamen>     |
+      | adellijke titel of predicaat (02.20) |                 |
+      | voorvoegsel (02.30)                  | <voorvoegsel>   |
+      | geslachtsnaam (02.40)                | <geslachtsnaam> |
+      | aanduiding naamgebruik (61.10)       | <naamgebruik>   |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | naam                                 | waarde                  |
+      | adellijke titel of predicaat (02.20) |                         |
+      | voorvoegsel (02.30)                  | <partner voorvoegsel>   |
+      | geslachtsnaam (02.40)                | <partner geslachtsnaam> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                            |
+      | type                | RaadpleegMetBurgerservicenummer   |
+      | burgerservicenummer | 000000164                         |
+      | fields              | adressering.gebruikInLopendeTekst |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                  | waarde                  |
+      | gebruikInLopendeTekst | <gebruikinlopendetekst> |
+
+      Voorbeelden:
```

Comment:
Wellicht in dit voorbeeld in enkele regels de kolom 'geslacht' de waarde 'M' geven.

Reply:
1 keer een M toegevoegd